api/serializers/__init__.py

```diff
@@ -105,7 +105,39 @@ class Meta:
         ]
 
 
-class ProfileSerializer(serializers.ModelSerializer):
+class StrictReadOnlyFieldsMixin:
+    """Raises a validation error (400) if read only fields are in the body of PUT/PATCH requests.
```

Comment:
```suggestion
    """
    Raises a validation error (400) if read only fields are in the body of PUT/PATCH requests.
```

Reply:
(suggestion - blocking) Let's utilize similar code style where the multi-line comment `"""` starts on a new line when there is multiple lines of comment while it is on the same line as `"""` when there is only one line of comment.